src/app/core/store/sdr/sdr.effects.ts

```diff
@@ -720,7 +707,18 @@ export class SdrEffects {
                   } else {
                     sidebarItem.queryParams.filters = sdrFacet.field;
                   }
+
+                  selectedFilterValues.forEach((selectedFilterValue: string) => {
+                    filterValue += `,${selectedFilterValue}`;
```

Comment:
This is fine if you are certain there is always something to the left.

I am imagining that if there is only a selectedFilterValue and nothing else then there will be an initial comma incorrectly applied.

This is simply a concern of mine.

Reply:
Is a valid concern. I will double check before merging.